### ISSUE TITLE
Add min. Craft version to README.md

### ISSUE THREAD

**Issue by tomholland** [engram-design/SuperTable#39]
getFieldsForElementsQuery in SuperTable_BlockElementType needs at least Craft v2.3.2615 (http://buildwithcraft.com/updates#build2615) - the plugin throws SQL errors otherwise.

Might be worth adding that requirement to README.md (with a large amount of custom additions (and the re-testing required after each upgrade), updating Craft regularly isn't always an option on some sites).

**Comment by engram-design** [comment]
An excellent point, and thanks for raising this! This has been added to the readme, and enforced with the next round of updates to the plugin.

**Comment by tomholland** [comment]
I'm not sure if there'll be other methods / etc. used in the plugin which need a higher version than v2.3.2615; I just figured out that getFieldsForElementsQuery was added in v2.3.2615 (and that's what was causing SQL errors for me). If we upgrade the site currently using the old Craft version, I suspect we'll try and make the leap straight to the latest Craft version - so won't be able to give a more accurate indicator of what the minimum Craft version is, I'm afraid.

It might be safest to put the suggested minimum version as whatever version you currently test on - with a note that versions below v2.3.2615 are known not to work, and versions in-between are untested.
